proto/p4/p4runtime.proto

```diff
@@ -300,14 +300,14 @@ message CounterEntry {
 // counters is not allowed and will return an error. The insertion/deletion
 // should happen as part of INSERT/DELETE on the associated table-entry.
 // For ReadRequest, the scope is defined as follows:
-// - All counter cells for all meters if counter_id = 0 (default).
-// - All counter cells for given counter_id if table_entry.match is empty.
+// - All counter cells for all tables if table_entry.table_id = 0.
+// - All counter cells of a table if table_entry.table_id is present and
+//   table_entry.match is empty.
 message DirectCounterEntry {
-  uint32 counter_id = 1;
   // The associated table entry match.
```

Comment:
suggest: "The associated table entry. This field is required."

Reply:
+1, and same for `DirectMeterEntry`